modules/registry-configuring-registry-storage-rhodf-cephrgw.adoc

```diff
@@ -70,19 +70,14 @@ $ AWS_SECRET_ACCESS_KEY=$(oc get secret -n openshift-storage rgwtest -o yaml | g
 $ oc create secret generic image-registry-private-configuration-user --from-literal=REGISTRY_STORAGE_S3_ACCESSKEY=${AWS_ACCESS_KEY_ID} --from-literal=REGISTRY_STORAGE_S3_SECRETKEY=${AWS_SECRET_ACCESS_KEY} --namespace openshift-image-registry
```

Comment:
@BlaineEXE Image registry must use image-registry-private-configuration-user with REGISTRY_STORAGE_S3_ACCESSKEY and REGISTRY_STORAGE_S3_SECRETKEY keys, it can't recognize keys from ObjectBucketClaim secret directly

Reply:
I'll leave it up to your team to decide whether to add support for other secrets or not then. Be aware that something like key rotation on the bucket (currently a manual step, and very rare) would render the image registry broken until its own secret is manually updated.